### PR TITLE
New package: Alteri.Avalon.Client version 2024.3.1

### PR DESCRIPTION
**Pull request by pythion**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161194)

### CODE REVIEW

**Comment by pythion** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @pythion,

Could you add a `PackageUrl`?

File: manifests/a/Alteri/Avalon/Client/2024.3.1/Alteri.Avalon.Client.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+PackageIdentifier: Alteri.Avalon.Client
+PackageVersion: 2024.3.1
+PackageLocale: en-US
+Publisher: Alteri
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: Alteri
+PackageName: Alteri Avalon
+PackageUrl: https://alteri.verymad.net/avaloninstall.php
```
Comment (stephengillie) [inline manifests/a/Alteri/Avalon/Client/2024.3.1/Alteri.Avalon.Client.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://alteri.verymad.net/
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @pythion,

The `PackageUrl` should be a website eventually leading to the `InstallerUrl`.

File: manifests/a/Alteri/Avalon/Client/2024.3.1/Alteri.Avalon.Client.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+PackageIdentifier: Alteri.Avalon.Client
+PackageVersion: 2024.3.1
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+Installers:
```
Comment (stephengillie) [inline manifests/a/Alteri/Avalon/Client/2024.3.1/Alteri.Avalon.Client.installer.yaml:12]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @pythion,

This package seems to have a dependency:
image

**Pull request closed without merging**